jwala-integration-test/src/test/resources/com/cerner/jwala/ui/selenium/configuration/generateAndDeployResources.feature

```diff
@@ -0,0 +1,108 @@
+Feature: Upload Resource - Jvm Node
+
+  Scenario:Jvm Node
+    Given I logged in
+    And I am in the configuration tab
+    And I created a group with the name "seleniumGroup"
+    And I created a media with the following parameters:
+      | mediaName       | jdk1.8.0_92     |
+      | mediaType       | JDK             |
+      | archiveFilename | jdk1.8.0_92.zip |
+      | remoteDir       | d:/ctp          |
+    And I created a media with the following parameters:
+      | mediaName       | apache-tomcat-7.0.55     |
+      | mediaType       | Apache Tomcat            |
+      | archiveFilename | apache-tomcat-7.0.55.zip |
+      | remoteDir       | d:/ctp                   |
+    And I created a media with the following parameters:
+      | mediaName       | apache-httpd-2.4.20     |
+      | mediaType       | Apache HTTPD            |
+      | archiveFilename | apache-httpd-2.4.20.zip |
+      | remoteDir       | d:/ctp                  |
+    And I created a jvm with following parameters:
+      | jvmName    | seleniumJvm          |
+      | tomcat     | apache-tomcat-7.0.55 |
+      | jdk        | jdk1.8.0_92          |
+      | hostName   | usmlvv2cto0766       |
+      | portNumber | 9000                 |
+      | group      | seleniumGroup        |
+    And I created a webserver with following parameters:
+      | webserverName      | seleniumWebserver   |
+      | hostName           | usmlvv2cto0766      |
+      | portNumber         | 80                  |
+      | httpsPort          | 443                 |
+      | group              | seleniumGroup       |
+      | apacheHttpdMediaId | apache-httpd-2.4.20 |
+      | statusPath         | /apache_pb.png      |
+
+    And I created a webapp with following parameters:
+      | webappName  | seleniumWebapp |
+      | contextPath | /hello         |
+      | group       | seleniumGroup  |
+
+    And I am in the resource tab
+#    upload webserver resources
+    And I expanded component "seleniumGroup"
+    And I expanded component "Web Servers"
+    And I clicked on component "seleniumWebserver"
+    And I clicked on add resource
+    And I fill in the "Deploy Name" field with "httpd.conf"
+    And I fill in the "Deploy Path" field with "d:\ctp\app\data\httpd\seleniumWebserver"
+    And I choose the resource file "httpdconf.tpl"
+    And I click the upload resource dialog ok button
```

Comment:
We might be able to get away without a "when" but we should follow Cucumber scenario best practices and follow Given-When-Then for consistency